src/components/journeys/JourneyInstancesDataTable/index.tsx

```diff
@@ -43,29 +46,40 @@ const JourneyInstancesDataTable: FunctionComponent<JourneysDataTableProps> = ({
 
   return (
     <>
-      <UserConfigurableDataGrid
+      <DataGridPro
         checkboxSelection
         columns={columnsWithHeaderTitles}
         components={{ Toolbar: Toolbar }}
         componentsProps={{
           toolbar: {
             gridColumns: columnsWithHeaderTitles,
             setQuickSearch,
-            setSortModel,
-            sortModel,
+            setSortModel: modelGridProps.onSortModelChange,
+            sortModel: modelGridProps.sortModel,
           },
         }}
         disableSelectionOnClick={true}
+        filterModel={modelGridProps.filterModel}
+        onColumnOrderChange={(params) => {
+          setColumnOrder(params.colDef.field, params.targetIndex - 1);
+        }}
+        onColumnResize={(params) => {
+          setColumnWidth(params.colDef.field, params.width);
+        }}
+        onFilterModelChange={(model) => {
+          if (!isEqual(model, modelGridProps.filterModel)) {
```

Comment:
Is this logic necessary? I would have thought it would be enough to just pass `modelGridProps.onFilterModelChange` directly to the property of the grid. Indeed, I would have thought passing them all at once using `...gridProps` would be enough.

Reply:
As we spoke about, I will look at what it does and how it works when removing this!